apps/sms/services/js/message_manager.js

```diff
@@ -41,28 +54,54 @@ var MessageManager = {
     );
   },
 
+  /**
+   * Event handler executed when a message is about to be 
+   * sent.
+   * @param {object} e The event object.
```

Comment:
> Wouldn't all methods be private?

Nope, just methods and properties that are used only by the object they it belongs to.

> How am I supposed to find out?

You can search within sms folder to check if any other object uses these methods/properties directly (like views/conversation/conversation.js uses services/js/message_manager.js sendSMS). But please ignore unit tests, sometimes we use private methods inside tests, we shouldn't do this though, but do sometimes when it's not possible to test important functionality in a nicer way.